src/domain/organization/constants.tsx

```diff
@@ -77,3 +111,16 @@ export const ORGANIZATION_SELECT_FIELDS = [
   ORGANIZATION_FIELDS.CLASSIFICATION,
   ORGANIZATION_FIELDS.PARENT_ORGANIZATION,
 ];
+
+export const MERCHANT_TEXT_FIELD_MAX_LENGTH = {
+  [WEB_STORE_MERCHANT_FIELDS.BUSINESS_ID]: 9,
+  [WEB_STORE_MERCHANT_FIELDS.CITY]: 50,
+  [WEB_STORE_MERCHANT_FIELDS.EMAIL]: 254,
+  [WEB_STORE_MERCHANT_FIELDS.NAME]: 100,
+  [WEB_STORE_MERCHANT_FIELDS.PAYTRAIL_MERCHANT_ID]: 100,
+  [WEB_STORE_MERCHANT_FIELDS.PHONE_NUMBER]: 18,
+  [WEB_STORE_MERCHANT_FIELDS.STREET_ADDRESS]: 500,
+  [WEB_STORE_MERCHANT_FIELDS.TERMS_OF_SERVICE_URL]: 500,
```

Comment:
```suggestion
  [WEB_STORE_MERCHANT_FIELDS.TERMS_OF_SERVICE_URL]: 200,
```
This particular `URLField` uses the default `max_length`.

Reply:
Good catch. Fixed the field max length